### ISSUE TITLE
clarification about metadata accessible to keybase.io staff

### ISSUE THREAD

**Issue by mossy-nw** [keybase/client#15301]
Is there any difference between (top level) Teams and Subteams with respect to metadata that Keybase servers (need to) have access to?  

From [your Teams announcement](https://keybase.io/blog/introducing-keybase-teams) I understand keybase.io servers know team names, membership, and roles but is this also true for subteams?  The [Teams high level design docs](https://keybase.io/docs/teams/design) indicate 
> The very existence of subteams are hidden from all who aren't members of the subteam. 
but I am guessing "all" here does not include Keybase servers (since membership of subteams would be needed by Keybase servers for message routing), correct?  Could you please clarify?

Finally, you've stated that #channel names are encrypted and therefore hidden from Keybase servers.  Is membership of (subscribers to) a chat #channel known to Keybase servers? 

Thanks for pulling together such a well-designed team chat client and being so thoughtful about building in contingencies in the event your servers become compromised.  I'm recommending keybase to more and more people all of the time, and I appreciate any clarity you can offer (maybe a `Metadata Matrix` to complement the awesome `Access Matrix` at https://keybase.io/docs/teams/design ?)

**Comment by maxtaco** [comment]
Correct, it's access-control (not encryption) that keeps subteam names private, so it's visible to the Keybase servers (and therefore keybase staff) for the reasons you give.

Membership to chat #channels is known to our servers (and to staff) for sending push notifications, computing badging, etc.

Thanks for your thoughtful questions!
